wls-basisdaten-service/src/main/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/exception/ExceptionConstants.java

```diff
@@ -31,4 +31,13 @@ public class ExceptionConstants {
     public static ExceptionDataWrapper POSTHANDBUCH_SPEICHERN_NICHT_ERFOLGREICH = new ExceptionDataWrapper("316",
             "postHandbuch: Das speichern des Handbuches war nicht erfolgreich.");
 
+    public static ExceptionDataWrapper GETUNGUELTIGEWAHLSCHEINE_PARAMETER_UNVOLLSTAENDIG = new ExceptionDataWrapper("331",
+            "getUngueltigews: Suchkriterien unvollständig.");
+    public static ExceptionDataWrapper GETUNGUELTIGEWAHLSCHEINE_KEINE_DATEN = new ExceptionDataWrapper("332", "Die Ungueltigews konnte nicht geladen werden.");
```

Comment:
```suggestion
    public static ExceptionDataWrapper GETUNGUELTIGEWAHLSCHEINE_KEINE_DATEN = new ExceptionDataWrapper("332", "Die ungültigen Wahlscheine konnten nicht geladen werden.");
```

Reply:
oder 
```suggestion
    public static ExceptionDataWrapper GETUNGUELTIGEWAHLSCHEINE_KEINE_DATEN = new ExceptionDataWrapper("332", "Die UngueltigeWahlscheine konnte nicht geladen werden.");
```
falls das im Bezug auf das Objekt gemeint ist.

Reply:
fixed mit 4a45d291993d6f3561550cd3df6733c3a8f44329

Lösung: `Die ungueltigen Wahlscheine konnte nicht geladen werden.`